AWS-Transcription.py

```diff
@@ -1,86 +1,102 @@
-from __future__ import print_function
+#!usr/bin/env python3
+
 import boto3
 import time
-import os
+from distutils.util import strtobool
+from pathlib import Path
 
 #Sets variables
 s3 = boto3.client('s3')
-print('Enter the s3 bucket name')
-s3Bucket = input()
-
-print('Enter name of file in Audio to upload')
-fileName = input()
-filePath = 'Audio/' + fileName
```

Comment:
The old filepath technically works, but it's not portable and would break if the code is run from elsewhere, hence why I replaced it with `pathlib.Path` and anchored it to the script itself.